### PR TITLE
Small improvements in secure service sample

### PR DESCRIPTION
**Pull request by oyvindronningstad**
- Allow services to be disabled.
 - Print FICR.INFO.VARIANT

### CODE REVIEW

File: samples/nrf9160/secure_services/src/main.c [hunk 1]
```diff
@@ -88,16 +88,30 @@ void main(void)
 #endif
 
 	u32_t info_part = 0;
-	u32_t ficr_addr = (u32_t)&NRF_FICR_S->INFO.PART;
+	u32_t ficr_info_part_addr = (u32_t)&NRF_FICR_S->INFO.PART;
 
-	printk("\nRead FICR, offset 0x20C (address 0x%08x):\n", ficr_addr);
-	ret = spm_request_read(&info_part, ficr_addr, sizeof(u32_t));
+	printk("\nRead FICR, offset 0x20C (address 0x%08x):\n",
```
Comment (hakonfam) [inline samples/nrf9160/secure_services/src/main.c]: These two calls look repetitive, would it make sense to extract a function for this?

```
ficr_read(u32_t addr, char *offset, char *name)
```

Comment (oyvindronningstad) [reply samples/nrf9160/secure_services/src/main.c]: Done

**Comment by pkchan** [comment]
Looks good. Just one more question:
Is CONFIG_SPM_SERVICE_READ supposed to be a service we can disable in this application?

**Comment by oyvindronningstad** [comment]
> Is `CONFIG_SPM_SERVICE_READ` supposed to be a service we can disable in this application?

I thought about it and decided against it, just to avoid noise in the code.

**Comment by oyvindronningstad** [comment]
Ready to merge

**Comment by hakonfam** [comment]
Check if the changelog should be updated

**Pull request merged**